packages/quantic/check-sfdx-project.js

```diff
@@ -0,0 +1,24 @@
+const pathlib = require('path');
+const fs = require('fs');
+
+function getGrandParentPath(path) {
+  const directories = path.split(pathlib.sep);
+  return directories.slice(0, directories.length - 2).join(pathlib.sep);
```

Comment:
So this expect to be in `./node_modules/Quantic` ?

Could be a good idea to add this as a comment for future simplicity if this every breaks?

Reply:
yeah exactly! I'll add comments to improve this 👌